### ISSUE TITLE
Migrate to asset list v2

### ISSUE THREAD

**Issue by p0mvn** [osmosis-labs/sqs#103]
We should migrate to asset list v2.

https://github.com/osmosis-labs/assetlists/blob/main/osmosis-1/generated/frontend/assetlist.json

**Comment by JeremyParish69** [comment]
Context on the Osmosis Labs' Assetlist improvement effort (and migration to a new file) dubbed 'Assetlistv2':

Osmosis Zone Frontend app uses the osmosis-labs/assetlists/ repo to keep a record of assets listed on Osmosis Zone and to serve the frontend app, which reads files from this repo directly. The frontend is the primary motivator for the assetlist, and no other apps are considered when establishing requirements or designing the assetlist.

It started with the ('old'/'assetlist v1') at https://github.com/osmosis-labs/assetlists/blob/main/osmosis-1/osmosis-1.assetlist.json. This served the frontend reasonably well, but there was still much asset-related data that still had to be hardcoded directly into the frontend repo, which is a hinderance to the asset listing process. At a certain point, a new file became the best option to best to serve the many and complex asset-related data needed by the frontend.

As of ~Feb 2021, the Osmosis Zone frontend app has switched to instead look at a new assetlist file; the ('new'/'current') 'assetlist v2' is at https://github.com/osmosis-labs/assetlists/blob/main/osmosis-1/generated/frontend/assetlist.json.
There are some notable differences: It sits in a new location; a directory called /generated/, which ideally signals to contributors that this file should only be generated--never modified directly--and also sits in a directory called /frontend/ to help disambiguate from the identically-titled assetlist.json files which are used in the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry/blob/master/osmosis/assetlist.json). Most notably, the file follows a different schema; the top level structure--being asset objects within an `assets` array--is the same, but each asset object has different (mostly more) properties to better handle the complex demands of the Osmosis Zone app--features that previously had to be hardcoded into the frontend repo.

The (old) Assetlist v1 is currently still retained and maintained because some other apps, tools, interfaces, may have established dependencies on it, and we want to decommission it gracefully without major disruption to any apps.

SQS has become a vital service for various features of the app, providing and otherwise unattainable quality of performance. Thus, retaining its functionality is essential. However, it seems that SQS does have a dependency on the old Assetlist v1. As with any other app dependent on assetlist v1, it would be best to update it to instead look at assetlist v2 to ensure that it remains functional (and may eventually be able to make use of additional data it provides) after the assetlist v1 eventually is decommissioned.

TL;DR
Please switch SQS's dependency on Assetlist v1 to Assetlist v2.
Due to the immense importance of SQS: The old assetlist v1 will NOT be decommissioned until SQS no longer depends on it.

**Comment by p0mvn** [comment]
Closing in favor of https://github.com/osmosis-labs/sqs/issues/134
